OMGVA PoS/Data layer/Models/Discount.cs

```diff
@@ -0,0 +1,13 @@
+﻿using OMGVA_PoS.Data_layer.Enums;
+
+namespace OMGVA_PoS.Data_layer.Models
+{
+    public class Discount
+    {
+        public long Id { get; set; }
+        public short Amount { get; set; }
```

Comment:
Maybe we could use a decimal right here to match payments (since there we use decimal for amounts)?

Reply:
I am pretty sure that according to specification this is meant to represent a percentage. They used tinyint to represent this value.